internal/util/sessionutil/session_util.go

```diff
@@ -329,6 +335,15 @@ func (s *Session) getServerID() (int64, error) {
 	return nodeID, nil
 }
 
+func (s *Session) getServerLabelFromEnv(role string) string {
+	switch role {
+	case "querynode":
+		return os.Getenv("MILVUS_COMPONENT_LABEL")
```

Comment:
Restricting the format or value range of labels to make system easier to maintain